server/lib/queries.js

```diff
@@ -36,7 +36,7 @@ const getTotalDonations = () => {
     SELECT SUM(${generateFXConversionSQL()}) AS "totalDonationsInUSD"
     FROM "Transactions"
     WHERE amount > 0 AND "PaymentMethodId" IS NOT NULL
-  `.replace(/\n/g, ' '), // this is to remove the new lines and save log space.
+  `.replace(/\s\s+/g, ' '), // this is to remove the new lines and save log space.
```

Comment:
Are you sure? I tested it and it always replaces multiple spaces with a single space. 
![screenshot 2016-10-24 18 03 21](https://cloud.githubusercontent.com/assets/6106941/19665694/3ed4a812-9a14-11e6-9dbc-0ab883fb6381.png)